assets/src/edit-story/components/canvas/selectionCanvas.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+import useCanvas from '../canvas/useCanvas';
+import withOverlay from '../overlay/withOverlay';
+import InOverlay from '../overlay';
+
+const LasoMode = {
```

Comment:
Should probably by "lasso" instead of "laso"?

Reply:
:)

Reply:
Done